### ISSUE TITLE
🍏 About this dashboard modal

### ISSUE THREAD

**Issue by vogelino** [vogelino/data-2-resilience#22]
![Image](https://github.com/user-attachments/assets/21fbe620-b9da-4316-8599-8fbb344e3e79)

**Comment by alsino** [comment]
Perfect!

Maybe we could increase the modal padding a bit and also add these two pieces of information below:

- https://github.com/vogelino/data-2-resilience/issues/13
- https://github.com/vogelino/data-2-resilience/issues/12

**Comment by vogelino** [comment]
I agree about the padding. I will change that. Regarding the other info, let's takle them on their own tickets later. Ok?
